internal/controller/dnsrecord_controller.go

```diff
@@ -271,15 +282,15 @@ func (r *DNSRecordReconciler) deleteRecord(ctx context.Context, dnsRecord *v1alp
 	hadChanges, err := r.applyChanges(ctx, dnsRecord, managedZone, true)
 	if err != nil {
 		if strings.Contains(err.Error(), "was not found") || strings.Contains(err.Error(), "notFound") {
-			logger.Info("Record not found in managed zone, continuing", "dnsRecord", dnsRecord.Name, "managedZone", managedZone.Name)
+			logger.Info("Record not found in managed zone, continuing", "managedZone", managedZone.Name)
 			return false, nil
 		} else if strings.Contains(err.Error(), "no endpoints") {
-			logger.Info("DNS record had no endpoint, continuing", "dnsRecord", dnsRecord.Name, "managedZone", managedZone.Name)
+			logger.Info("DNS record had no endpoint, continuing", "managedZone", managedZone.Name)
 			return false, nil
 		}
 		return false, err
 	}
-	logger.Info("Deleted DNSRecord in manage zone", "dnsRecord", dnsRecord.Name, "managedZone", managedZone.Name)
+	logger.Info("Deleted DNSRecord in manage zone", "managedZone", managedZone.Name)
```

Comment:
Example:

```.json
{
  "level": "info",
  "ts": "2024-06-04T16:32:37+01:00",
  "logger": "dnsrecord_controller",
  "msg": "Deleted DNSRecord in manage zone",
  "controller": "dnsrecord",
  "controllerGroup": "kuadrant.io",
  "controllerKind": "DNSRecord",
  "DNSRecord": {
    "name": "t-single-lively-waterfall",
    "namespace": "dnstest"
  },
  "namespace": "dnstest",
  "name": "t-single-lively-waterfall",
  "reconcileID": "81a1838b-c782-4323-8baf-58486cadbcec",
  "managedZone": "dev-mz-aws"
}
```

Left `managedZone` here for now although i expect that will be moved and set on the context higher up the stack as part of https://github.com/Kuadrant/dns-operator/issues/117